webview/src/plots/components/templatePlots/TemplatePlotsGrid.tsx

```diff
@@ -63,16 +63,14 @@ export const TemplatePlotsGrid: React.FC<TemplatePlotsGridProps> = ({
     e.stopPropagation()
   }, [])
 
-  useEffect(() => {
-    const panels = document.querySelectorAll('.vega-bindings')
-    return () => {
-      for (const panel of Object.values(panels)) {
-        panel.removeEventListener('mouseenter', addDisabled)
-        panel.removeEventListener('mouseleave', removeDisabled)
-        panel.removeEventListener('click', disableClick)
-      }
-    }
-  }, [addDisabled, removeDisabled, disableClick])
```

Comment:
This was useless since there are no more panels at this stage. And because I moved the function, it dropped coverage as it is never called.